HEC.FDA.Model/LifeLoss/LifeSimDatabase.cs

```diff
@@ -0,0 +1,144 @@
+﻿using Statistics;
+using Statistics.Histograms;
+using System.Data.SQLite;
+using System.Text;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace HEC.FDA.Model.LifeLoss
+{
+    /// <summary>
+    /// Handles operations for querying .fia databases imported from LifeSim
+    /// </summary>
+    public class LifeSimDatabase
+    {
+        private string _connectionString;
+        private static string[] _lifelossColumns = {"Name", "Alternatives", "Time_2", "Time_4",
+                                                   "Time_6, Time_8", "Time_10", "Time_12",
+                                                   "Time_14", "Time_16", "Time_18", "Time_20", "Time_22"};
+
+        public LifeSimDatabase(string dbpath)
+        {
+            _connectionString = $"Data Source={dbpath}";
+        }
+
+        /// <summary>
+        /// Change the list of simulations available to the user to select from
+        /// </summary>
+        /// <returns></returns>
+        public List<LifeSimSimulation> UpdateSimulations()
+        {
+            StringBuilder sb = new();
+            sb.Append("SELECT ");
+            for (int i = 0; i < _lifelossColumns.Length; i++)
+            {
+                sb.Append(_lifelossColumns[i]);
+                if (i < _lifelossColumns.Length - 1)
+                    sb.Append(", ");
+            }
+            sb.Append(" FROM Simulations_Lookup_Table");
+            string query = sb.ToString();
+            List<LifeSimSimulation> simulations = [];
+
+            try
+            {
+                using SQLiteConnection connection = new SQLiteConnection(_connectionString);
+                connection.Open();
+
+                using var command = new SQLiteCommand(query, connection);
+                using var reader = command.ExecuteReader();
+                while (reader.Read())
+                {
+                    string name = reader.GetString(0); // Name is 0th selection
+                    LifeSimSimulation simulation = new LifeSimSimulation(name);
+
+                    simulation.Alternatives = reader.GetString(1).Split(',').ToList(); // Alternatives are 1st selection and in csv string format
+
+                    // Hazard Times are in the 2nd through 12th selections                                                           
+                    for (int i = 2; i <= 12; i++)
+                    {
+                        if (reader.GetBoolean(i))
+                            simulation.HazardTimes.Add($"{(i - 1) * 2}"); // convert time to integer string, starting from 2 and incrementing by 2 until 22
+                    }
+                    simulations.Add(simulation);
+                }
+            }
+            catch (Exception ex)
```

Comment:
What should we do with this exception? either we should just throw, or handle it somehow. What do you think?